style.css

```diff
@@ -54,6 +59,7 @@ ul {
     border: 1px solid #cc092f;
 	list-style: none;
 	text-transform: uppercase;
+    text-align: center;
```

Comment:
Not a fan of centred text on the home and about us pages. It does look good on the AUV, Gallery, Sponsor, and Contact pages.